client/src/views/header/HeaderPage.vue

```diff
@@ -92,7 +92,7 @@
 
           <profile-header
             id="profile-button"
-            :name="userInfo && userInfo.humanHandle ? userInfo.humanHandle.label : ''"
+            :name="(userInfo && userInfo.humanHandle.label) || ''"
```

Comment:
Isn't name converted to a boolean here? Shouldn't it be `userInfo ? userInfo.humanHandle.label : ''` instead?